specification/ai/Azure.AI.Projects/common/models.tsp

```diff
@@ -115,3 +116,23 @@ model AssetCredentialResponse {
   @doc("Credential info to access the storage account.")
   blobReference: BlobReference;
 }
+
+#suppress "@azure-tools/typespec-azure-core/no-string-discriminator"
+@doc("Abstract data class.")
+@discriminator("type")
+@added(Versions.v2025_05_15_preview)
+@removed(Versions.v1)
+model InputData {
```

Comment:
"InputData" is too generic. Please name it such that it implies what operation uses it.